### PR TITLE
Add basic readme.md for server side

### PR DESCRIPTION
**Pull request by Anmolbansal1**
Also fixed vulnerabilities issues in react npm package with `npm audit fix`

### CODE REVIEW

File: fact-bounty-server/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+> 
+
+## Setup
+
+### Pre-requisites
+* NodeJS
+* MongoDB 
+
+Copy .env.example to .env and make necessary changes to it.
```
Comment (rajdeepbharati) [inline fact-bounty-server/README.md]:
```suggestion
Copy `.env.example` to `.env` and specify environment variables.
```

File: fact-bounty-server/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Copy config/keys.example.js to config/keys.js and enter database address to it.
```
Comment (rajdeepbharati) [inline fact-bounty-server/README.md]:
```suggestion
Copy `config/keys.example.js` to `config/keys.js` and enter database address to it.
```

**Comment by Anmolbansal1** [comment]
@ivantha please review

**Comment by ivantha** [comment]
I am closing this as node server is now deprecated.

**Pull request closed without merging**